buildSrc/src/main/groovy/EhOsgi.groovy

```diff
@@ -76,7 +76,7 @@ class EhOsgi implements Plugin<Project> {
           }
         }
 
-        instruction 'Export-Package', '*'
+        instruction 'Export-Package', 'org.ehcache.core.internal.service', '*'
```

Comment:
That's too coarse grained, and will cause issues because multiple OSGi packages will end up exporting the same.
Check the `gradle.properties` found in specific modules - `core` here - and add the entry there.